docs/software/gaming.md

```diff
@@ -349,4 +350,46 @@ Make sure you have one of DietPi's File Servers installed.
 - Steam is installed to the DietPi user data directory `/mnt/dietpi_userdata/steam`
 - By default, installing games will install them to `/mnt/dietpi_userdata/steam`
 
+## PaperMC - Fast and Optimized Minecraft Server, Extendable with Plugins
+
+Descent 1 & 2. A masterpiece 3D FPS. Brought back to life with the DXX-Rebirth project. Play Descent as true to the original with OpenGL/ES rendering.
+
+- DietPi will install the Demo and Shareware versions of Descent. Please see the FAQ below to transfer the full game.
+- Keyboard + Mouse is highly recommended
+- We have compiled the latest version of DXX-Rebirth (0.58.1) with support for FB and RPi OpenGL.
+
+![DietPi gaming software DXX-Rebirth](../assets/images/dietpi-software-games-papermc.jpg){: style="width:500px"}
```

Comment:
Logo should be a bit smaller:  
{: style="width:500px"} -> {: style="width:150px"}

In other cases we also used 200px or also 100px.

Reply:
With the text included, it could stay at 500px? https://github.com/PaperMC/papermc.io/raw/master/src/images/logo-marker.svg

Ähm okay, the text is white, on white background invisible 😄. Should be possible to make it grey by adding `fill="#181a1c"` to the style.